lib/circuitry/tasks.rb

```diff
@@ -7,10 +7,10 @@
     logger = Logger.new(STDOUT)
     logger.level = Logger::INFO
 
-    Circuitry::Provisioning.provision(logger: logger)
-  end
+    if Rake::Task.task_defined?(:environment)
+      Rake::Task[:environment].invoke
```

Comment:
I think you might have it backwards: https://blog.simplificator.com/2014/12/30/rake-execute-vs-invoke/